.ci/azure-pipelines-build.yml

```diff
@@ -70,3 +70,59 @@ jobs:
           tags: |
             stable-$(Build.BuildNumber)-$(BuildConfiguration)
             $(Version)-$(BuildConfiguration)
+
+  - job: PushManifests
+    displayName: 'Build and Publish Docker Manifests'
+
+    pool:
+      vmImage: 'ubuntu-latest'
+
+    variables:
+      DOCKER_CLI_EXPERIMENTAL: enabled
+
+    steps:
+      - script: |
+          # sudo sed -i '$s/}/,\n"experimental": true\n}/' /etc/docker/daemon.json
+          echo '{"experimental": true}' | sudo tee /etc/docker/daemon.json
+          sudo systemctl restart docker
+          docker version
+        displayName: 'Enable Experimental Features in Docker Server'
```

Comment:
One of these seems superfluous. Do we need both?

Reply:
Ooh wait never mind. Server vs client. I'm surprised that sudo works.

Reply:
AFAIK yes as explained [here](https://gist.github.com/awesomebytes/16d0a5fcc4f368a43ac21e5cf3000034). First is for the client, second for the daemon.

Reply:
Regarding the [docker docs](https://docs.docker.com/engine/reference/commandline/manifest_push/) it's client experimental only - I'll remove it later.
In the other link I linked in the OP they enabled it because they use binfmt to build the multi-arch images if I remeber correctly - which we don't need.